Core/VariantManager.swift

```diff
@@ -42,12 +42,12 @@ public struct Variant {
     
     public static let defaultVariants: [Variant] = [
         // SERP testing
-        Variant(name: "sc", weight: doNotAllocate, features: []),
+        Variant(name: "sc", weight: 1, features: []),
         Variant(name: "sd", weight: doNotAllocate, features: []),
-        Variant(name: "se", weight: doNotAllocate, features: []),
+        Variant(name: "se", weight: 1, features: []),
         
-        Variant(name: "me", weight: 1, features: []),
-        Variant(name: "mf", weight: 1, features: [.tabSwitcherListLayout])
+        Variant(name: "me", weight: doNotAllocate, features: []),
+        Variant(name: "mf", weight: doNotAllocate, features: [.tabSwitcherListLayout])
```

Comment:
Could we add a filter property to the Variants to allow filters like "ipad", "EnglishOnly" etc to be set on a per-experiment basis? Check out the filter lambdas in https://github.com/duckduckgo/Android/blob/develop/app/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt. Happy to defer this if there isn't time right now.

Reply:
Yep, I was thinking that.  I'll need that for the iPad improvements project as that'll be going out at the same time or shortly after so I'll do it there as that shouldn't delay just getting this out.

Reply:
Perfect! It'll be handy to have it defined in the `Variant` for future experimentation. Please, include me in that PR, there are some surprising nuances to not changing the population distribution.